src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4638,4 +4638,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_ConstantStringTooLong" xml:space="preserve">
     <value>Length of String constant exceeds current memory limit.  Try splitting the string into multiple constants.</value>
   </data>
+  <data name="ERR_DebugEntryPointNotSourceMethodDefinition" xml:space="preserve">
+    <value>Debug entry point must be a definition of a source method in the current compilation.</value>
```

Comment:
@gafter We use the term "compilation" in other API error messages (such as "tree not part of compilation", etc.). I don't see any errors mentioning "project".